tetrateci/tsb-util/tsb-objects/serviceroute.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: traffic.tsb.tetrate.io/v2
+kind: ServiceRoute
+Metadata:
+  organization: {{ orgName }}
+  name: {{ serviceRouteName }}
+  group: {{ groupName }}
+  workspace: {{ workspaceName }}
+  tenant: {{ tenantName }}
+spec:
+  service: {{ hostFQDN }}
```

Comment:
this has to be of the form {{ ns }}/{{ hostFQDN }}

Reply:
Generated hostFQDN has back in it for reviews instead of 'mid'. Cross check

Reply:
reviews should be back right?